rfcs/rfc-team-notifications.md

```diff
@@ -0,0 +1,256 @@
+
+In the author's experience, MS Teams generates a lot notifications, by virtue of the many teams and chat groups subscribed to.
+
+### Zenhub
+
+This software is a "layer" on top of GitHub Issues and provide workspaces, sprint flows, pipelines, and various reports.
+
+Zenhub appears to support only basic local and email notifications for issue events, but as is it built upon GitHub, GitHub notifications can be used instead as they are more configurable.
+
+### GitHub
+
+This system stores BCGov repositories, personal repos, and provides branching, Pull Request (PR) flows and merging. This system also stores Issues, although most developers probably view issues in the Zenhub "view". GitHub supports lots of local and email notification options, including the concept of "subscriptions", which allows users to "watch" only the specific items they care about.
+
+In the author's experience, GitHub generates a lot of notifications, which can be received on the GitHub site (or CLI) or via email. The author recommends receiving emails ("push") instead of checking GitHub ("pull"). The author finds email notifications easy to temporarily ignore, and GitHub emails particularly quick to review and handle -- items that need follow-up can be left in the Inbox and other items can be promptly deleted.
+
+### Email
+
+This system allows for messaging between single or multiple individuals. This is a corporate system and every team member has access to it.
+
+Incoming emails generally display a notification, although filters or rules can be configured in most email clients to automatically dispatch certain emails, thus avoiding a notification.
+
+In the author's experience, email generates a lot of notifications, either about operations (eg, customer issue, specific design questions, meetings) or sent from another system (eg, GitHub). However, as mentioned in [GitHub](#github) above, emails seem to be easy to handle, likely because selected items can be left in the Inbox for later response (ie, as a kind of "to do list") instead of getting "lost" in a long chat stream.
+
+In the author's experience, the High / Low Importance flags on emails is used inconsistently and is therefore not useful for determining objective message priority. Internal guidance could be provided for better setting this flag, thereby adding a useful priority level to some emails, but since emails come from all different organizations, this is unlikely to be a general improvement. Therefore, the Importance flag is not discussed further in this RFC.
+
+Note that email is a good system to use for group discussions that include people who don't have access to MS Teams.
+
+Email is also a good system to use for discussions that should be saved (eg, in a folder) for possible later retrieval.
+
```

Comment:
Consider adding: Contractors should check their government e-mails on regular basis.

Reply:
Is this a given? Or can contractors get by without a government email account? (As far as I know, the only reason I need a gov email is to use MS Teams.)

Reply:
Contractors can only use summer.gov.bc.ca and it times out every 5 minutes or so. A contractor must remember to manually refresh the session or re-log in on a regular basis to check their gov email inbox.

Reply:
Yeah, it's a real pain.

Reply:
Olga, Yui, do you think this issue of using gov email belongs in this RFC? It sounds like a separate issue to me (though it's definitely an issue to be resolved, since we can't live without it, and it's cumbersome to live with it).

Reply:
Agreed - Notification via Email is a separate thing.